util/walk_test.go

```diff
@@ -0,0 +1,194 @@
+package util_test
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
```

Comment:
Since we usually keep compatibility to the two previous versions... can you avoid the usage of `io/fs`?

Reply:
sure! I will replace it with what is required